### PR TITLE
API Rules translations

### PR DESCRIPTION
**Pull request by Sawthis**
**Description**

Changes proposed in this pull request:

- ...
- ...
- ...

**Related issue(s)**

### CODE REVIEW

**Review by qbalukom — APPROVED** [review]

**Review by qbalukom — APPROVED** [review]

File: core/src/i18n/en.yaml [hunk 1]
```diff
@@ -3,9 +3,34 @@ addons:
...
+    messages:
+      one-method: This access strategy requires at least one method.
+    title: Access Strategies
+    tooltips:
+      delete: Define at least one access strategy.
+      path: Path can contain alphanumeric characters and  '/', '.', '*', '?', '!', '-', '(', and ')'.
+  buttons:
+    create: Create API Rules
+  general-settings:
+    title: General settings
```
Comment (NHingerl) [inline core/src/i18n/en.yaml:19]:
```suggestion
    title: General Settings
```

File: core/src/i18n/en.yaml [hunk 2]
```diff
@@ -35,19 +60,25 @@ common:
   created: Created
   delete-dialog:
     delete-confirm: Don't ask this again
+    information: You've switched off delete confirmations for all resources. You can switch them back on in Preferences.
```
Comment (NHingerl) [inline core/src/i18n/en.yaml:63]:
```suggestion
    information: You've switched off removal confirmations for all resources. You can switch them back on in Preferences.
```

As the dialog tile is "Remove..." we should keep using the word "remove".

**Review by qbalukom — APPROVED** [review]

**Pull request merged**